lib/providers/discord.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+exports = module.exports = function () {
+
+    return {
+        protocol: 'oauth2',
+        auth: 'https://discordapp.com/api/oauth2/authorize',
+        token: 'https://discordapp.com/api/oauth2/token',
+        scope: ['email', 'identify'], // see https://discordapp.com/developers/docs/topics/oauth2#scopes
+        profile: function (credentials, params, get, callback) {
+
+            get('https://discordapp.com/api/users/@me', null, (profile) => {
+
+                credentials.profile = {
+                    id: parseInt(profile.id),
```

Comment:
I am not sure I agree with the parseInt. If they return this id as a string we should honor that and not throw when they don't!